src/pages/about/index.tsx

```diff
@@ -14,73 +14,105 @@ const Explanation = () => {
       </h2>
       <div className="mx-auto max-w-4xl" style={{ textIndent: 50 }}>
-        <p className="my-4">A Defifa game happens in four stages:</p>
+        <p className="my-4">A Defifa game happens in five stages:</p>
 
         <ol>
           <li className="my-2">
-            <b>1. Create:</b> Anyone can create a Defifa game. Game creators
-            represent different predictions as NFTs, decide the price of each
-            NFT, and set the game's schedule. Game creators also have the option
-            to reserve a percentage of the final pot or a percentage of the NFTs
-            for their team, for NFT artists, or for someone else.
+            <b>1. Create:</b> Anyone can create a Defifa game. Game creators set
+            up teams, set the prices of each team's NFT, and set the game's
+            schedule. Game creators can choose to reserve a percentage of the
+            pot or a percentage of the NFTs for the recipients of their
+            choosing.
           </li>
           <li className="my-2">
-            <b>2. Mint:</b> People make their predictions by minting NFTs – one
-            person can mint multiple NFTs corresponding to multiple predictions.
-            After an amount of time set by the game's creator, NFT minting
-            closes.
+            <b>2. Mint:</b> People can join teams by minting their NFTs. One
+            person can mint multiple NFTs from multiple teams. After an amount
+            of time set by the game's creator, minting closes.
           </li>
           <li className="my-2">
-            <b>3. Score:</b> Once a game finishes, anyone can score the game's
-            results to determine how much of the pot will go to each
-            prediction's NFTs. At least 50% of the NFTs for at least 50% of the
-            predictions have to vote for a set of scores – otherwise, people can
-            reclaim their ETH from the pot.
+            <b>3. Refunds:</b> A refund window opens if the game's creator set
+            one up. During this window, anyone can redeem their NFT to reclaim
+            their ETH from the pot. <b>This stage is optional.</b>
           </li>
           <li className="my-2">
-            <b>4. Earn:</b> If a set of scores is approved, people will be able
-            to burn their NFTs to claim their share of the rewards based on
-            their predictions.
+            <b>4. Score:</b> Once a game ends, anyone can score the game's
+            results to determine how much of the pot will go to each team. At
+            least 50% of teams need to approve a set of scores by majority vote
+            – otherwise, ETH stays in the pot.
+          </li>
+          <li className="my-2">
+            <b>5. Earn:</b> Once scores are approved, people can burn their NFTs
+            to claim a portion of their team's ETH from the pot. They also get a
+            portion of any Defifa/Juicebox governance tokens earned by the game.
```

Comment:
is it both dififa and jbx here? or just defifa?

Reply:
Both (see [this message](https://discord.com/channels/1040017278302965761/1070415506777063444/1121530134189576264)).